[CI] DocsClientYamlTestSuiteIT test {yaml=reference/rest-api/usage/line_38} failing

### CI Link

https://gradle-enterprise.elastic.co/s/k5ttr7y6ybz6w

### Repro line

gradlew ':docs:yamlRestTest' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/rest-api/usage/line_38}" -Dtests.seed=B33396E7A6E1CE74 -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dtests.locale=da-DK -Dtests.timezone=Asia/Omsk

### Does it reproduce?

Yes

### Applicable branches

main

### Failure history

https://gradle-enterprise.elastic.co/scans/tests?search.timeZoneId=Europe/Bucharest&tests.container=org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT&tests.test=test%20%7Byaml%3Dreference/rest-api/usage/line_38%7D

### Failure excerpt

Similar to https://github.com/elastic/elasticsearch/issues/95603, but not the same output in the `usage`. Now complaining about `analytics_collections: <unexpected> but was <{count=0}>`.

```
REPRODUCE WITH: ./gradlew ':docs:yamlRestTest' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/rest-api/usage/line_38}" -Dtests.seed=CB9D9AB30CC180F4 -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dtests.locale=lv-LV -Dtests.timezone=America/Argentina/Salta -Druntime.java=20

org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT > test {yaml=reference/rest-api/usage/line_38} FAILED
    java.lang.AssertionError: Failure at [reference/rest-api/usage:16]: Expected a map containing
                         watcher: a map containing
                         execution: a map containing
                             actions: a map containing
                                  _all: a map containing
                                   total: <6>
                        total_time_in_ms: <164>
                             watch: a map containing
                               input: a map containing
                                  _all: a map containing
                                   total: <0>
                                  active: <0>
                             trigger: a map containing
                                  _all: a map containing
                                   total: <0>
                                  active: <0>
                         available: <true>
                             count: a map containing
                               total: <0>
                              active: <0>
                           enabled: <true>
                             eql: a map containing
                         available: <true>
                           enabled: <true>
                          features: a map containing
                               joins: a map containing
                    join_queries_three: <0>
                      join_queries_two: <0>
                            join_until: <0>
             join_queries_five_or_more: <0>
                     join_queries_four: <0>
                            sequence: <6>
                                keys: a map containing
                         join_keys_two: <0>
                         join_keys_one: <4>
                       join_keys_three: <0>
                join_keys_five_or_more: <0>
                        join_keys_four: <0>
                                join: <0>
                           sequences: a map containing
                sequence_queries_three: <1>
                 sequence_queries_four: <0>
                  sequence_queries_two: <5>
                        sequence_until: <1>
                      sequence_maxspan: <3>
         sequence_queries_five_or_more: <0>
                               event: <18>
                               pipes: a map containing
                             pipe_tail: <21>
                             pipe_head: <0>
                           queries: a map containing
                                 all: a map containing
                                 total: <24>
                                failed: <0>
                                _all: a map containing
                                 total: <24>
                                failed: <0>
               enterprise_search: a map containing
               search_applications: a map containing
                               count: <0>
                         available: <true>
                           enabled: <true>
             analytics_collections: <unexpected> but was <{count=0}>
                  frozen_indices: a map containing
                     indices_count: <0>
                         available: <true>
                           enabled: <true>
                         archive: a map containing
                     indices_count: <0>
                         available: <true>
                           enabled: <true>
                             ilm: a map containing
....
```

--- a/docs/reference/rest-api/usage.asciidoc
+++ b/docs/reference/rest-api/usage.asciidoc
@@ -413,6 +413,9 @@ GET /_xpack/usage
     "enabled": true,
     "search_applications" : {
       "count": 0
+    },
+    "analytics_collections": {
+      "count": 0
     }
   }
 }

